etcdserver/server.go

```diff
@@ -167,6 +168,8 @@ type EtcdServer struct {
 
 	SyncTicker <-chan time.Time
 
+	// transport used to get version information from peers
+	trans    *http.Transport
```

Comment:
I want its name different from the rafthttp.transport in raftNode struct. I could call it `versionTrans` to make it more descriptive.